core/sys/windows/user32.odin

```diff
@@ -489,3 +500,30 @@ WINDOWINFO :: struct {
 	wCreatorVersion: WORD,
 }
 PWINDOWINFO :: ^WINDOWINFO
+
+MIIM_STATE      :: 0x00000001
+MIIM_ID         :: 0x00000002
+MIIM_SUBMENU    :: 0x00000004
+MIIM_CHECKMARKS :: 0x00000008
+MIIM_TYPE       :: 0x00000010
+MIIM_DATA       :: 0x00000020
+
+MIIM_STRING :: 0x00000040
+MIIM_BITMAP :: 0x00000080
+MIIM_FTYPE  :: 0x00000100
+
+MENUITEMINFOW :: struct {
+    cbSize: UINT,
+    fMask: UINT,
+    fType: UINT,         // used if MIIM_TYPE (4.0) or MIIM_FTYPE (>4.0)
+    fState: UINT,        // used if MIIM_STATE
+    wID: UINT,           // used if MIIM_ID
+    hSubMenu: HMENU,      // used if MIIM_SUBMENU
+    hbmpChecked: HBITMAP,   // used if MIIM_CHECKMARKS
+    hbmpUnchecked: HBITMAP, // used if MIIM_CHECKMARKS
+    dwItemData: ULONG_PTR,   // used if MIIM_DATA
+    dwTypeData: LPWSTR,    // used if MIIM_TYPE (4.0) or MIIM_STRING (>4.0)
+    cch: UINT,           // used if MIIM_TYPE (4.0) or MIIM_STRING (>4.0)
+    hbmpItem: HBITMAP,      // used if MIIM_BITMAP
```

Comment:
Make sure to use tabs rather than spaces.

Reply:
Try to rebase up I think most of the changes here are present in master now ;)